app/views/tasklist/show.html.erb

```diff
@@ -1,4 +1,4 @@
-<% content_for :title, "Learn to drive a car: step by step" %>
+<% content_for :title, tasklist[:title] %>
 
 <% content_for :meta_tags do %>
   <meta name="description" content="Learn to drive a car in the UK - get a provisional licence, take driving lessons, prepare for your theory test, book your practical test.">
```

Comment:
We need to fix this too.  We should probably get it from the description field of the content item that we publish in [collections-publisher](https://github.com/alphagov/collections-publisher/blob/release/lib/tasks/publishing_api.rake#L52) as that's where Rummager will get it from.  There's a `ContentItem` model that might be useful to help retrieve that.

(We also need to get some better words to put in those descriptions from Jon, but that's another story)